include/swift/Driver/Job.h

```diff
@@ -162,8 +169,9 @@ class CommandOutput {
   StringRef getPrimaryOutputFilename() const;
 
   /// Return a all of the outputs of type \c getPrimaryOutputType() associated
-  /// with a primary input. Note that the returned \c StringRef vector may be
-  /// invalidated by subsequent mutations to the \c CommandOutput.
+  /// with a primary input. The return value will contain one \c StringRef per
+  /// primary input, _even if_ the primary output type is TY_Nothing, and the
+  /// primary output filenames are therefore all empty strings.
```

Comment:
I'd feel better if you noted that this was considered incorrect behavior.

Reply:
Done.